deployments/helm/proxy-nsmgr/templates/proxy-nsmgr.tpl

```diff
@@ -1,17 +1,17 @@
 ---
 apiVersion: apps/v1
-kind: DaemonSet
+kind: Deployment
 metadata:
   name: proxy-nsmgr
   namespace: {{ .Release.Namespace }}
 spec:
```

Comment:
For deployments you should add a `replicas` item to the spec so the helm chart can deploy more than 1 replica, e.g. `replicas: {{ .Values.replicaCount }}`

Reply:
also, add a default value to values.yaml for `replicaCount`

Reply:
Thank you for the comments, Tim! The issues you mentioned are now fixed.